src/viewport.js

```diff
@@ -133,9 +133,17 @@ $.Viewport = function( options ) {
         animationTime:   this.animationTime
     });
 
+    this.degreesSpring = new $.Spring({
+        initial: options.degrees,
+        springStiffness: this.springStiffness,
+        animationTime: this.animationTime
+    });
+    delete options.degrees;
+
     this._oldCenterX = this.centerSpringX.current.value;
     this._oldCenterY = this.centerSpringY.current.value;
     this._oldZoom    = this.zoomSpring.current.value;
+    this._oldDegree  = this.degreesSpring.current.value;
```

Comment:
I think this should be called `_oldDegrees`.

Reply:
good catch!